obstacle_detector: 
  {
    max_range: 2.0,
    min_range: 0.03,
    obstacles_frame: "laser",
    visualization_frame: "base_link",
    output_frame: "base_link",
    lidar_offset: 0.0,
    segment_threshold: 0.03,
    max_segment_size: 0.5,
    min_segment_size: 0.04,
    min_to_divide: 0.03,
    upside_down: false
  }
